spec/helpers/hyrax_helper_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rspec'
+
+RSpec.describe HyraxHelper, type: :helper do
+  describe "#capitalize_value" do
```

Comment:
It looks like you changed the method name to `titleize_value` above so these tests need to be updated to match.